frontend/common/src/components/UserProfile/ProfileSections/WorkPreferencesSection.tsx

```diff
@@ -26,45 +26,107 @@ const WorkPreferencesSection: React.FunctionComponent<{
     : null;
 
   const anyCriteriaSelected = !isEmpty(acceptedOperationalArray);
+
+  // all V2 operational requirements
+  const operationalRequirementsSubsetV2 = [
+    OperationalRequirement.OvertimeOccasional,
+    OperationalRequirement.OvertimeRegular,
+    OperationalRequirement.ShiftWork,
+    OperationalRequirement.OnCall,
+    OperationalRequirement.Travel,
+    OperationalRequirement.TransportEquipment,
+    OperationalRequirement.DriversLicense,
+  ];
+
+  // requirements that have not been selected made into an array
+  const unselectedOperationalArray = operationalRequirementsSubsetV2.filter(
+    (requirement) => !acceptedOperationalRequirements?.includes(requirement),
+  );
+
+  // generate list of unaccepted operational requirements
+  const unacceptedOperationalArray = unselectedOperationalArray
+    ? unselectedOperationalArray.map((opRequirement) => (
+        <li data-h2-font-weight="b(700)" key={opRequirement}>
+          {opRequirement
+            ? getOperationalRequirement(opRequirement).defaultMessage
+            : ""}
+        </li>
+      ))
+    : null;
+
   return (
     <div
       data-h2-bg-color="b(lightgray)"
       data-h2-padding="b(all, m)"
       data-h2-radius="b(s)"
     >
-      {anyCriteriaSelected && (
+      {wouldAcceptTemporary === null && (
         <p>
+          <p>
```

Comment:
Can we remove on of the `<p>` here so we don't nest them?

Reply:
tackled in [this](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3405/commits/268a3d7742ca12e773ce6faa3793156204ca656e)

Reply:
You are going to hate me, but i think the wrong tag was removed leading to a string not being wrapped in the `<p>`

Reply:
hopefully all good now